plugin/storage/kafka/options.go

```diff
@@ -34,6 +34,12 @@ const (
 	EncodingProto = "protobuf"
 	// EncodingZipkinThrift is used for spans encoded as Zipkin Thrift.
 	EncodingZipkinThrift = "zipkin-thrift"
+	// EncodingZipkinProto is used for spans encoded as Zipkin Protobuf.
```

Comment:
Same with these comments - can you make it clear they are only relevant for otel images.